src/components/Title.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from "@testing-library/react";
+import Title from "./Title";
+
+describe("Title", () => {
+  it("shows the title of a page", () => {
+    render(<Title pageName={"Suche"} />);
+
+    expect(screen.getByText("Suche")).toBeInTheDocument();
```

Comment:
You could search by the role of `heading` here. Even by a heading with the level of `1`. You can find out about it in the testing-library documentation.